tests/console/kdump_and_crash.pm

```diff
@@ -37,6 +37,13 @@ sub run {
     $self->wait_boot;
     select_console 'root-console';
 
+    if (check_var('ARCH', 'ppc64le') || check_var('ARCH', 'ppc64')) {
+        if (script_run('kver=$(uname -r); kconfig="/boot/config-$kver"; [ -f $kconfig ] && grep ^CONFIG_RELOCATABLE $kconfig')) {
+            record_soft_failure 'poo#49466 -- No kdump if no CONFIG_RELOCATABLE in kernel config';
```

Comment:
Since this will basically stop the module from continuing further, just update the mention and mention that... To avoid bugs filed against [a feature](https://openqa.opensuse.org/tests/930182#step/kdump_and_crash/48) :D. Another idea is to return 1 instead. The module will be marked as a soft failure anyways.

Reply:
Hi Santiago, do you want me to change the comment passed to record_soft_failure ? I thought current one was explicit enough to say that kdump would not be started as running kernel was not configured for that.

Reply:
@michelmno No, just returning 1 would do the trick (So that "The result that openqa can't display" will not confuse anybody), I have retriggered your job: https://openqa.opensuse.org/tests/933698

But also please be aware of https://lists.opensuse.org/opensuse-factory/2019-05/msg00124.html

Reply:
I thought the uknown error would go away :/, but anywho. Looks fine on my side.

Reply:
I am triggering clone-custom as new https://openqa.opensuse.org/tests/933734 waiting for completion as your 933703 is incomplete (based on previous snapshot)

Reply:
https://openqa.opensuse.org/tests/933703 soft failed :) exactly in kdump_and_crash, anyway let's wait still I guess... as they won't be deployed anyway for the time being